### PR TITLE
Set Permission for Upload folder

### PR DESCRIPTION
**Pull request by crizbe**
https://sourceforge.net/p/phpipam/support-requests/191/

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -66,5 +66,8 @@ RUN cp ${WEB_REPO}/config.dist.php ${WEB_REPO}/config.php && \
     -e "s/\['port'\] = 3306;/\['port'\] = 3306;\n\n\$password_file = getenv(\"MYSQL_ENV_MYSQL_PASSWORD_FILE\");\nif(file_exists(\$password_file))\n\$db\['pass'\] = preg_replace(\"\/\\\\s+\/\", \"\", file_get_contents(\$password_file));/" \
     ${WEB_REPO}/config.php
 
+#set upload folder permission
+RUN chmod 777 /var/www/html/app/subnets/import-subnet/upload
```
Comment (pierrecdn) [inline Dockerfile]: As you can see just above https://github.com/pierrecdn/phpipam/pull/29/files#diff-3254677a7917c6c01f55212f86c57fbfR61, the rights are already given to `www-data` (Apache httpd) in this directory.
Why would you need to allow 0777 perms here? Something else that Apache has to be able to read/write?

**Review by pierrecdn — CHANGES_REQUESTED** [review]
Hi,
Please provide more context.
Thanks,

File: Dockerfile (hunk 1 above)
Comment (pierrecdn) [reply Dockerfile]: Also, you don't want to create an additional container layer just for that, you have to embed into the previous RUN statement.

Comment (crizbe) [reply Dockerfile]: To Upload Ip Adresses with a .csv file the upload folder needs 777 permissions (as descriped in the link above), otherwhise there will be the following error: "Cannot move file to upload dir".

Comment (pierrecdn) [reply Dockerfile]: Hi,

I still don't understand why you would get an error with www-data being the owner of this directory (as per https://github.com/pierrecdn/phpipam/pull/29/commits/473110dd4981ef3b6fb3cf203ca36be180d8a458#diff-3254677a7917c6c01f55212f86c57fbfR60).
I know that Miha answered that in this thread, but I would like more analysis to be perfomed.

I've personally no issue at importing VLANs/IPs and so on without doing any change.

**Comment by pierrecdn** [comment]
Hi,
I've personally no issue at importing VLANs/IPs and so on without doing any change.
Could you give a reproducer?

**Comment by crizbe** [comment]
Hi pierrecdn,
The Issue has been fixed with IPAM version 1.3.2. It's no longer necessary to change the permission.
Regards cbxxen

**Pull request closed without merging**